docs/openshift-installation.md

```diff
@@ -0,0 +1,18 @@
+# Openshift Installation
+
+This installation method is the recommended way for Openshift users. **Note**: Openshift version must be 4.2 or higher.
+
+The NGINX Ingress Operator is a [RedHat certified Operator](https://connect.redhat.com/en/partner-with-us/red-hat-openshift-operator-certification).
+
+1. In the Openshift dashboard, click in `Operators` > `Operator Hub` in the left menu and use the search box to type `nginx ingress`:
+![alt text](./images/openshift1.png "Operators")
+1. Click in the NGINX Ingress Operator and click Install:
+![alt text](./images/openshift2.png "NGINX Ingress Operator")
+1. Click in Subscribe (**Note:** We recommend to mark Specific Namespace for the Operator):
+![alt text](./images/openshift3.png "NGINX Ingress Operator Install")
+
+Openshift will install the NGINX Ingress Operator:
+
+![alt text](./images/openshift4.png "NGINX Ingress Operator Subscribe")
+
+You can now deploy the NGINX Ingress Controller instances manually following the [examples](../examples) or using the Openshift Dashboard.
```

Comment:
Is there any link for OpenShift dashboard?

Not sure if "manually" should be used. Does the OpenShift dashboard provide some kind of  non-manual way? 🤔 

```suggestion
You can now deploy the NGINX Ingress Controller instances [using `kubectl`](../examples) or using the Openshift Dashboard.
```

Reply:
What do you mean by link? To a documentation? or in a cluster? In a cluster it depends on the fqdn of the installation.

Yes, the way of adding CRs in Openshift is using the Openshift dashboard (instead of oc/kubectl). Everything through the dashboard. Makes sense?

Reply:
yeah, a link to some documentation (ours or external). 

because it wasn't clear what you meant by "using the Openshift Dashboard".

or perhaps we should not mention the dashboard at all, because we don't provide any documentation for that?

Reply:
Ok will remove the Dashboard part. I think most OS customers are used to do this over the dashboard anyways, so we'll keep this open